src/methods/broadcast_tx_commit.rs

```diff
@@ -1,3 +1,76 @@
+//! Sends blocking transactions.
+//!
+//! Sends a signed transaction to the RPC and waits until transaction is fully complete.
+//!
+//! ## Example
+//!
+//! ```
```

Comment:
Might also help to include a link to an example that shows how to use this method. The `contract_change_method` shows how to use `broadcast_tx_async`, may I ask you to introduce an example that uses the `broadcast_tx_commit` method and link to it?

Reply:
I've created a new example that uses the `broadcast_tx_commit` method and opened a separate [PR](https://github.com/near/near-jsonrpc-client-rs/pull/86) for it.